lighthouse-core/report/v2/renderer/details-renderer.js

```diff
@@ -15,20 +15,85 @@
  */
 'use strict';
 
-/* globals self */
+/* globals self Util */
+
+/**
+ * Create render context for critical-request-chain tree display.
+ * @param {!DetailsRenderer.CRCNode} tree
+ * @return {!{tree: DetailsRenderer.CRCNode, startTime: number, transferSize: number}}
+ */
+function initTree(tree) {
```

Comment:
> what do you think about sticking these inside the _renderCrc? feels weird to pollute global scope with them

maybe we should make a new file...it would be good to have a lot more context on some of this, like what are `treeMarkers`, etc :)